Makefile

```diff
@@ -1,4 +1,8 @@
-.PHONY: test
+.PHONY: init test
+
+init:
+	luarocks init --no-gitignore
```

Comment:
Would be great to add the `luarocks --local install nlua` here also. And shouldn't the busted install be local also?